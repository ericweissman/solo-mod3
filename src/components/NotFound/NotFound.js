import React from 'react'
import { Link } from 'react-router-dom'

const NotFound = () => {
  return (
    <div>
      <h4>Dagnabbit! This page doesn't exist!</h4>
      <Link to='/'>
        <button>Back To Safety</button>
      </Link>
    </div>
  )
}

export default NotFound