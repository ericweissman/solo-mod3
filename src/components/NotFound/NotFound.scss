.not-found {
  @extend .card;
  margin: auto;
  width: 300px;
  height: 160px;
  margin: 100px auto;
  p {
    color: $text-white;
    font-family: $secondary-font;
  }
}