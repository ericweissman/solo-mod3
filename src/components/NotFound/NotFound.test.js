import React from 'react'
import { shallow } from 'enzyme'
import NotFound from '../NotFound/NotFound'

describe('NotFound', () => {
  let wrapper
  it('should match the correct snapshot', () => {
    wrapper = shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot()
  })
})