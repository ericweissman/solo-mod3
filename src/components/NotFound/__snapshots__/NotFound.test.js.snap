// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match the correct snapshot 1`] = `
<div
  className="not-found"
>
  <h4>
    Dagnabbit!
  </h4>
  <p>
    This page doesn't exist!
  </p>
  <Link
    replace={false}
    to="/"
  >
    <button>
      Back To Safety
    </button>
  </Link>
</div>
`;
